### PR TITLE
Update Dicom docs to use tensorflow-io (vs. tensorflow-io-nightly)

### PR DESCRIPTION
**Pull request by yongtang**
As dicom is already part of the versioned release.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: docs/tutorials/dicom.ipynb [hunk 1]
```diff
@@ -104,7 +104,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/tree/master/tensorflow_io/dicom) in TensorFlow IO to decode DICOM files with TensorFlow."
+        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/blob/master/tensorflow_io/core/python/ops/dicom_ops.py) in TensorFlow IO to decode DICOM files with TensorFlow."
```
Comment (lamberta) [inline docs/tutorials/dicom.ipynb]: Can we point to https://www.tensorflow.org/io/api_docs/python/tfio/dicom
When this is published on the site? @MarkDaoust

Comment (yongtang) [reply docs/tutorials/dicom.ipynb]: Thanks @lamberta, the PR has been updated.

**Comment by yongtang** [comment]
Ping @MarkDaoust @lamberta, the PR has been rebased and updated. Would you mind take another look and see if everything is ok?

File: docs/tutorials/dicom.ipynb [hunk 2]
```diff
@@ -105,7 +105,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/tree/master/tensorflow_io/dicom) in TensorFlow IO to decode DICOM files with TensorFlow."
+        "This tutorial shows how to use [decode_dicom_image](https://www.tensorflow.org/io/api_docs/python/tfio/image/decode_dicom_image) in TensorFlow IO to decode DICOM files with TensorFlow."
```
Comment (lamberta) [inline docs/tutorials/dicom.ipynb]: This should autolink when you wrap the API symbol in backticks: \`tfio.image.decode_dicom_image\`

**Comment by yongtang** [comment]
@lamberta Thanks! The PR has been updated.

**Pull request merged**